openedx_events/content_authoring/data.py

```diff
@@ -136,3 +151,31 @@ class CertificateConfigData:
     title = attr.ib(type=str)
     signatories = attr.ib(type=List[CertificateSignatoryData], factory=list)
     is_active = attr.ib(type=bool, default=False)
+
+
+@attr.s(frozen=True)
+class ContentLibraryData:
+    """
+    Data about changed ContentLibrary.
+
+    Arguments:
+        library_key (LibraryLocatorV2): a key that represents a Blockstore-based content library.
+        update_blocks (bool): flag that indicates whether the content library blocks indexes should be updated
```

Comment:
Why is this flag part of the ContentLibrary data?

Reply:
@mariajgrimaldi ^

Reply:
Thanks for the detailed explanation! If this info is beneficial to receivers, then let's leave it as is. Thanks again!